pkg/logutil/log.go

```diff
@@ -159,7 +180,13 @@ func (f *textFormatter) Format(entry *log.Entry) ([]byte, error) {
 	if file, ok := entry.Data["file"]; ok {
 		fmt.Fprintf(b, "%s:%v:", file, entry.Data["line"])
 	}
-	fmt.Fprintf(b, " [%s] %s", entry.Level.String(), entry.Message)
+	if f.DisableColors {
+		fmt.Fprintf(b, " [%s] %s", entry.Level.String(), entry.Message)
+	} else {
+		colorStr := logTypeToColor(entry.Level)
+		s := "\033" + colorStr + "m[" + entry.Level.String() + "] " + entry.Message + "\033[0m"
```

Comment:
Can we start the color before timestamp and end color before `\n`?

Reply:
Of course.